crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -190,8 +190,9 @@ def transition(inst, name, state)
         return [404, "Node not found"]
       end
 
-      if state == "readying"
-        transition_to_readying inst, name, state, node
+      if state == "ready" && !node.admin?
+        # we need to claim raids when we already have ohai data, i.e. after first chef-client run
+        process_raid_claims node
```

Comment:
I was thinking about that too after writing my comment. I guess we really need to focus on that bug :/